core/src/services/compfs/backend.rs

```diff
@@ -63,6 +63,19 @@ impl Builder for CompfsBuilder {
                 "root is not specified",
             )),
         }?;
+
+        // If root dir is not exist, we must create it.
```

Comment:
does not

Reply:
Hi, it's a requirement for all storage backend to make root dir available.

Reply:
No I'm saying it should be "if root dir **does** not exist, ..."

Reply:
Oh, got it.